### PR TITLE
[RLlib documentation] Enhance "RLlib in 60sec" example.

### PR DESCRIPTION
**Pull request by sven1977**
Enhance "RLlib in 60sec" example.
- Add stop criterium.
- Mention --no-ray-ui flag in case user has no dashboard installed.

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [x] This PR is not tested :(

### CODE REVIEW

**Comment by worldveil** [comment]
This doesn't use simple optimizer -- will this fix the GPU issue I was seeing?

or was I tagged b/c you want me to test this?

**Comment by sven1977** [comment]
No, it's not addressing your GPU issue. I wasn't able to reproduce this on any system I tested your repro on: Mac and Linux.
Also haven't seen issues with pandas not being found.

File: doc/source/rllib.rst [hunk 1]
```diff
@@ -31,17 +36,23 @@ Then, you can try out training in the following equivalent ways:
...
-                                           # --config='{"framework": "tf2", "eager_tracing": true}' for eager,
-                                           # --torch to use PyTorch OR --config='{"framework": "torch"}'
+  rllib train --run=PPO --env=CartPole-v0 --stop='{"episode_reward_mean": 180.0}' --no-ray-ui
+      # -v [-vv] for verbose,
+      # --config='{"framework": "tf2", "eager_tracing": true}' for eager,
+      # --config='{"framework": "torch"}' to use PyTorch.
+
+Note that the `--no-ray-ui` tells the train script to run without the Ray dashboard.
+This option has been (soft) deprecated as of Ray 1.7 and you should now use the more
+explicit `--ray-ui` flag instead (in case you would like to run with the dashboard).
```
Comment (richardliaw) [inline doc/source/rllib.rst:46]: hmm, why not just have the example show `--ray-ui`? Otherwise aren't you showing a deprecated flag?

Comment (sven1977) [reply doc/source/rllib.rst:46]: Showing `--no-ray-ui` would make the example work for any version of RLlib (also older ones).

Comment (richardliaw) [reply doc/source/rllib.rst:46]: But it's a deprecated flag. If the user wants to use an older version of Rllib, shouldn't they read the correct version of the docs?

**Pull request closed without merging**